liquibase-standard/src/main/java/liquibase/analytics/configuration/AnalyticsArgs.java

```diff
@@ -29,7 +26,7 @@ public class AnalyticsArgs implements AutoloadedConfigurations {
         ConfigurationDefinition.Builder builder = new ConfigurationDefinition.Builder("liquibase.analytics");
 
         ENABLED = builder.define("enabled", Boolean.class)
-                .setDescription("Enable or disable sending product usage data and analytics to Liquibase. Learn more at https://docs.liquibase.com/analytics. DEFAULT: true for OSS users | false for PRO users")
+                .setDescription("Enable or disable sending product usage data and analytics to Liquibase. Learn more at https://docs.liquibase.com/analytics.")
```

Comment:
It might be useful to provide a default value here, even if it is just symbolic. This way users won't be surprised to learn that on is the default.

Reply:
Resolved that I'm going to keep this as is over introducing drift in the pro repository